go/protoutil/duration_test.go

```diff
@@ -80,3 +90,40 @@ func TestDurationFromProto(t *testing.T) {
 		})
 	}
 }
+
+func TestDurationToProto(t *testing.T) {
+	t.Parallel()
```

Comment:
I think the `t.Parallel()` is cargo culted here from the existing tests, but these tests are so fast that I think we should just drop it everywhere.

Reply:
Removed `t.Parallel()` from this file